cookbook/integrations/kubernetes/kfmpi/mpi_mnist.py

```diff
@@ -66,7 +66,7 @@ def training_step(images, labels, first_batch, mnist_model, loss, opt):
     retries=3,
     cache=True,
     cache_version="0.1",
-    requests=Resources(cpu='1', mem="2Gi"),
+    requests=Resources(cpu='1', mem="600Mi"),
```

Comment:
why this change?

Reply:
https://github.com/flyteorg/flytesnacks/runs/6026259179?check_suite_focus=true - resulted in this error. Samhita had changed it for testing pyflyte run